_pages/tourney-rules.html

```diff
@@ -6,117 +6,109 @@
 ---
 
 <div class="container">
```

Comment:
```suggestion
<div class="container pt-3">
```
and remove the `<br />` on the line below.